docs/HowToArchiveJournal.md

```diff
@@ -17,13 +17,134 @@ For more info on NuoDB database journaling, please consult the official [NuoDB d
 
 Since the `journal` has to write all commits durably to disk, the speed of the disk directly influences the commit latency.
 To achieve the best performance, NuoDB recommends placing the `journal` on the fastest disk available.
+In contract, the `archive` should be using the disk with the largest throughput.
 
```

Comment:
Line 12, 
When what comes back online, the database, a lost process? I think the phrase is probably referring to the database. Maybe consider changing: 

when coming back online.
   to 
during the database startup procedure.